pages/ksphere/kommander/1.1.0-beta/backup-and-restore/index.md

```diff
@@ -8,4 +8,4 @@ category: K-Sphere
 menuWeight: 11
 ---
 
-Kommander stores all data within Kubernetes etcd and can be backed up and restored using the [standard back up and restore documentation](/ksphere/konvoy/latest/backup) in the Konvoy documentation. No special consideration must be made for Kommander when backing up or restoring Konvoy.
+Kommander stores all data as CRDs in the Kubernetes API and can be backed up and restored using the [standard Konvoy documentation](/ksphere/konvoy/1.5.0-beta/backup/). No special consideration must be made for Kommander when backing up or restoring Konvoy.
```

Comment:
```suggestion
Kommander stores all data, as CRDs, in the Kubernetes API and can be backed up and restored using the [standard Konvoy documentation](/ksphere/konvoy/1.5.0-beta/backup/). No special consideration must be made for Kommander when backing up or restoring Konvoy.
```

Reply:
For this example, do we explain, "coin" what CRD's are, maybe earlier in the doc? Or is CRD very well known?

Reply:
Seems like CRDs are well known enough for someone researching backup stuff.